sdk/servicebus/azure-servicebus/azure/servicebus/management/_utils.py

```diff
@@ -44,3 +75,168 @@ def get_next_template(list_func, *args, **kwargs):
             )
         )
     return feed_element
+
+
+def deserialize_value(value, value_type):
+    if value_type in ("int", "long"):
+        value = int(value)
+    elif value_type == "boolean":
+        value = value.lower() == "true"
+    elif value_type == "double":
+        value = float(value)
+    elif value_type == "dateTime":
+        value = isodate.parse_datetime(value)
+    elif value_type == "duration":
+        value = isodate.parse_duration(value)
```

Comment:
We should leave a note that if using times > than a month, this will return an isodate type, which is a bit... strange.

Reply:
Added an issue #12704 to track replacing isodate